lib/elixir/test/elixir/enum_test.exs

```diff
@@ -265,6 +265,20 @@ defmodule EnumTest do
     end
   end
 
+  test "drop/2 with streams" do
+    drop_stream = fn list, count -> list |> Stream.map(& &1) |> Enum.drop(count) end
```

Comment:
For now, I went with the same approach as `dedup/1 with streams`, but maybe in the longer run we should have a test case for streams for each function, like `EnumTest.Range`.